resources/public/api/conf/1.0/docs/organisations-details-self-assessment-description.md

```diff
@@ -0,0 +1,5 @@
+<p>Use a matchId to get an organisation's Self Assessment details. This includes turnover and tax solvency status.</p>
+
+<p>This endpoint will return available data for the most recent 4 years before the current Self Assessment tax year (CY-4).</p>
+
+<p>Self Assessment data may not be available for the most recent tax year because of the deadlines for sending tax returns. For example, the deadline for sending a tax return for the 2019-20 tax year was 31 October 2020 for a paper return, and 31January 2021 for sending online.</p>
```

Comment:
missing a space here
```suggestion
<p>Self Assessment data may not be available for the most recent tax year because of the deadlines for sending tax returns. For example, the deadline for sending a tax return for the 2019-20 tax year was 31 October 2020 for a paper return, and 31 January 2021 for sending online.</p>
```